reference/glossary.md

```diff
@@ -9,6 +9,10 @@ Exceptionally, some *actuators* may also be *sensors* if they make sensory measu
 
 **Ancestor**: When referring to a *node*, the *ancestor* of a *descendant* is a *node* which hierarchically contains the *descendant* in the *scene tree*, at any relative depth.
 
+**Base node**: A *base node* is a node a Webots built-in node.
```

Comment:
This sentence looks wrong.

Reply:
Probably `a node` should be removed, right?

Reply:
Yes...